recipes/csp-gateway/recipe.yaml

```diff
@@ -0,0 +1,71 @@
+context:
+  name: csp-gateway
+  version: 0.0.1
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+    url: https://pypi.org/packages/source/c/csp-gateway/csp_gateway-${{ version }}.tar.gz
+    sha256: 6c93db8664841568833811556fa18d190214ee1cf02cc5a71ee10d3a588f964f
+
+build:
```

Comment:
Looking at the `pyproject.toml` there are a number of script entry points that should be included and tested:

```
  85   │ [project.scripts]
  86   │ cspgateway-start = "csp_gateway.server.cli:main"
  87   │ cspgateway_start = "csp_gateway.server.cli:main"
  88   │ csp-gateway-start = "csp_gateway.server.cli:main"
  89   │ csp_gateway_start = "csp_gateway.server.cli:main"
```